tools/distrib/python/submit.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import shutil
+import subprocess
+import tempfile
+
+parser = argparse.ArgumentParser(
+    description='Submit the package to a PyPI repository.')
+parser.add_argument(
+    '--repository', metavar='r', type=str, default='pypi',
+    help='The repository to push the package to. '
+         'Ensure the value appears in your .pypirc file. '
+         'Defaults to "pypi".'
+)
+args = parser.parse_args()
+
+# Move to the directory of this file.
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+# Move to the root directory of Python GRPC.
+os.chdir('../../../src/python/src')
+
+# Make the push.
+cmd = 'python setup.py sdist upload -r %s' % args.repository
```

Comment:
Yeah, don't use 'sdist upload'. It's insecure. Use twine: https://pypi.python.org/pypi/twine